cognite_toolkit/cognite_modules/infield/cdf_infield_location/transformations/tr_asset_oid_workmate_infield_sync_assets_from_hierarchy_to_apm.yaml

```diff
@@ -10,6 +10,7 @@ destination:
 ignoreNullFields: true
 shared: true
 action: upsert
+dataSetExternalId: ds_infield_{{default_location}}_app_data
```

Comment:
So, this is not correct now. This is not the app data data set, but rather the default location asset data set, ds_asset_{{default_location}}_source_data if I recall correctly. But this is something @erikangelsen-cognite probably should quality assure.

Reply:
@gregertw yes you are correct, assets is ingested from their source system, so it should end up in the "source data" spaces and data sets.